spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -70,3 +70,50 @@ definitions:
             desc: |
               PRN-1 identifier of the satellite signal for which
               acquisition was attempted
+
+ - MSG_ACQ_SV_PROFILE:
+    id: 0x001E
+    short_desc: Reacq perfomance measurement and debug
+    desc: |
+      The message describes SV profile during acquisition time.
+      The message is useful to debug and measure the performance of
+      the new (re-)acq design.
+    public: true
+    fields:
+        - N:
```

Comment:
yeah it would make sense to send multiple messages since each acq event is independent of the other events